homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -175,24 +170,25 @@ async def async_create_remote_control(self, during_setup=False):
         except (URLError, SOAPError, OSError) as err:
             _LOGGER.debug("Could not establish remote connection: %s", err)
             self._control = None
-            self.state = STATE_OFF
+            self.state = MediaPlayerState.OFF
             self.available = self._on_action is not None
         except Exception:
             _LOGGER.exception("An unknown error occurred")
             self._control = None
-            self.state = STATE_OFF
+            self.state = MediaPlayerState.OFF
             self.available = self._on_action is not None
 
-    async def async_update(self):
+    async def async_update(self) -> None:
         """Update device data."""
         if self._control is None:
             await self.async_create_remote_control()
             return
 
         await self._handle_errors(self._update)
 
-    def _update(self):
+    def _update(self) -> None:
         """Retrieve the latest data."""
+        assert self._control is not None
         self.muted = self._control.get_mute()
         self.volume = self._control.get_volume() / 100
```

Comment:
`volume` is defined as `int`, the result here is a `float`. Technically that's fine as `int` is a subclass of `float`. It might just be better though to just define it as `float` instead.